apps/web/public/static/locales/en/common.json

```diff
@@ -1843,6 +1843,7 @@
   "review_event_type": "Review Event Type",
   "looking_for_more_analytics": "Looking for more analytics?",
   "looking_for_more_insights": "Looking for more Insights?",
+  "filters": "Filters",
```

Comment:
I can also add this field in some other language's common.js files if needed so let me know.

Reply:
No it's fine. you only have to add this key value in /en/common.json. we'll add to other languages later

Reply:
Ok

I have added another field:
```dif
"all_event_types_filter_label": "All Event Types",
```